### PR TITLE
Auth and Wishlist functionality and UI

### PR DESCRIPTION
**Pull request by ganesh-kanchi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,16 +4,26 @@
...
-    "miragejs": "^0.1.43",
-    "mockman-js": "^1.1.5",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "react-router-dom": "^6.3.0",
+    "miragejs": "^0.1.41",
+    "mockman-js": "^1.0.5",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^6.2.2",
```
Comment (paarthbajaj) [inline package.json:14]: what was the reason behind changing versions?

Comment (ganesh-kanchi) [reply package.json:14]: I struggled to get the mock api to work which is why I tried changeing everything I guess at last it did work.

**Review by paarthbajaj — APPROVED** [review]
The code looks good. Seen good practices. Remove unnecessary spaces from your code. Good efforts! You can merge.

File: src/requests/wishlistRequests/addToWishList.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+const addToWishList = (product, token) => {
+    return axios.post(
+        "/api/user/wishlist/",
+        { product },
+        { headers: { authorization: token } }
+    )
+};
+
+export { addToWishList };
```
Comment (paarthbajaj) [inline src/requests/wishlistRequests/addToWishList.js:11]: Won't be there too many files if you keep making different files for each function call? May I know the source of this inspiration?

Comment (ganesh-kanchi) [reply src/requests/wishlistRequests/addToWishList.js:11]: felt like I should oversimplify the code anyway it's in a different folder.

Comment (paarthbajaj) [reply src/requests/wishlistRequests/addToWishList.js:11]: Ok

**Review by Avin008 — APPROVED** [review]
code is well structured and readable, good to merge

**Pull request merged**